evals/scripts/campaign_evals_questions_ideals.py

```diff
@@ -47,19 +47,25 @@
         "question": "What are the top three genres that fans listen to outside of hip hop?",
         "ideal": "genres",
         "eval_name": "get_top_genres",
-        "criteria": sub_set_criteria
+        "criteria": subset_criteria
        
     },
     {
         "question": "What is the country distribution of fans?",
-        "ideal": "format: [country_name]: [fans_count], example: US: 123",
+        "ideal": "Example: \n• US:\n\t 121",
         "eval_name": "get_country_distribution",
-        "criteria": 'The actual answer should be formatted in the ideal value format.'
+        "criteria": "The answer should list of elements including country name, fans count like ideal value"
```

Comment:
Could we move this `criteria` to the shared `criteria` file for more consistent management of these `criteria`?